packages/bulkapi/src/database/LionWebQueries.ts

```diff
@@ -98,9 +89,11 @@ export class LionWebQueries {
         }
     }
 
-    createPartitions = async (partitions: LionWebJsonChunk): Promise<QueryReturnType<CreatePartitionsResponse>> => {
+    createPartitions = async (clientId: string, partitions: LionWebJsonChunk): Promise<QueryReturnType<CreatePartitionsResponse>> => {
         logger.dbLog("LionWebQueries.createPartitions")
-        const query = this.context.queryMaker.dbInsertNodeArray(partitions.nodes)
+        let query = `SET repo.version = ${getRepoVersion()};\n`
+        query += this.context.queryMaker.dbInsertNodeArray(partitions.nodes)
+
         // eslint-disable-next-line
```

Comment:
Can you disable ESLint for a specific warning that occurs?

Reply:
Done